CODEOWNERS

```diff
@@ -14,15 +14,16 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 /inference-engine/  @openvinotoolkit/openvino-ie-maintainers
 
 # IE CPU:
-/inference-engine/src/mkldnn_plugin/  @openvinotoolkit/openvino-ie-cpu-maintainers
-/inference-engine/include/cpu/  @openvinotoolkit/openvino-ie-cpu-maintainers
-/inference-engine/src/low_precision_transformations/  @openvinotoolkit/openvino-ie-cpu-maintainers
-/inference-engine/thirdparty/mkl-dnn/  @openvinotoolkit/openvino-ie-cpu-maintainers
+/inference-engine/src/mkldnn_plugin/  @openvinotoolkit/openvino-ie-cpu-maintainers @openvinotoolkit/openvino-ie-cpu-developers
+/inference-engine/include/cpu/  @openvinotoolkit/openvino-ie-cpu-maintainers @openvinotoolkit/openvino-ie-cpu-developers
```

Comment:
do we have such folder?

Reply:
Why you just can't add more people to maintainers group instead of adding developers with the same rights (as I understand)?

Reply:
@ilya-lavrenov Nope, we don't. I don't know why it has been added.

@evgenytalanin-intel Developers are supposed to have only "write" access, so they will be added automatically to PRs and will be able to approve it, but only maintainers can merge. At least that's the idea.

Reply:
@ilya-lavrenov I removed this folder

Reply:
@vladimir-paramuzov It was me who added that, I mis-read ie/include/cp**p** as cp**u**

Reply:
@ilya-lavrenov @evgenytalanin-intel Please resolve the discussion if you're OK with the changes and merge PR.